src/components/DefaultKeyboardAvoidingView.tsx

```diff
@@ -29,7 +29,7 @@ export const DefaultKeyboardAvoidingView = ({ children }: { children: React.Reac
   return (
     <KeyboardAvoidingView
       enabled={device.platform === 'ios'}
-      behavior={device.platform === 'ios' ? 'padding' : undefined}
+      behavior={device.platform === 'ios' ? 'padding' : 'height'}
```

Comment:
does this change behaviors of other forms, not SUE?

Reply:
I updated the behaviour to undefined again for Android. I will make 'height' specially for SUE.

https://github.com/smart-village-solutions/smart-village-app-app/pull/1032/commits/85ed935a1d40cdca70e84db2cad132376ad1c7c2